docs/tools/hedgehog-engine/rangers/models.md

```diff
@@ -7,4 +7,10 @@ description: Model tools for Sonic Frontiers
 Command line tool for working with Sonic Frontiers model files.
 
 [Download :fontawesome-solid-download:](https://github.com/HedgeDocs/HedgeDocs.github.io/releases/download/CompiledTools/HedgeNeedle.7z){ .md-button .md-button--primary}
-[Source Code :fontawesome-brands-github:](https://github.com/Radfordhound/HedgeLib/tree/HedgeLib%2B%2B/HedgeTools/HedgeNeedle){ .md-button target="_blank"}
\ No newline at end of file
+[Source Code :fontawesome-brands-github:](https://github.com/Radfordhound/HedgeLib/tree/HedgeLib%2B%2B/HedgeTools/HedgeNeedle){ .md-button target="_blank"}
+
+
+## HedgeEdit
+Editor tool, is used to edit material for frontiers.
+
+[Download :fontawesome-solid-download:](https://drive.google.com/file/d/1VDHTpPe1MNVlAYblLsDd3Oa1KpsLqLGs/view){ .md-button .md-button--primary}
```

Comment:
2 things:

- Please do not use Google Drive links for tools. Either use the assets system that's described in the README.md of HedgeDocs, or link directly to the tools' page, or link directly to the tool's page
- Material tools should not be in the modelling category. You should create a new page for material tools